frontend/src/semantic-ui/site/collections/breadcrumb.overrides

```diff
@@ -0,0 +1,3 @@
+/*******************************
```

Comment:
Will we be removing the empty files in the build?

Reply:
@julesfll Do you have an idea on how to remove empty files in build? Im not sure if we want to delete all these files, we might need it in the future if we decided to use more components from semantic ui